harrt_init init
---
single_objective_path[] paths
